doc/examples.rst

```diff
@@ -44,10 +44,12 @@ using running examples in FBS2e.
 
    cruise
    describing_functions
+   interconnect_tutorial
```

Comment:
We should probably update the introductory test at the top of this section since the Jupyter notebooks in this section are no longer using running examples in FBS2e (true before this change, but I noticed it now).  How about something like:

> These notebooks demonstrate the use of modeling, anaylsis, and design tools
using examples from textbooks ([FBS2e](https://fbswiki.org/wiki/index.php?title=FBS), [OBC](https://fbswiki.org/wiki/index.php?title=OBC)), courses, and other sources.

(links would have to be done differently, put using GitHub markup here to show where the links go).

Reply:
sounds good. INcorporated the suggested rewording. 

Also improved the name on one of the examples and switched to lists for specifying set of interconnected systems according to upcoming changes in `interconnect`.